std/cpp/Int32.hx

```diff
@@ -19,6 +19,7 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
  */
- package cpp;
+
+package cpp;
```

Comment:
Why ? just after the initial (c) block seems fine to me.

Reply:
Agreed, there's an open issue for this here: https://github.com/HaxeCheckstyle/haxe-formatter/issues/310

Reply:
So if I understand correctly, this is configurable, but the default is currently set to 1 line?

Reply:
Sort of... but changing the setting also removes empty lines between multiline comments (https://github.com/HaxeCheckstyle/haxe-formatter/issues/292). I'd want both, which isn't possible yet.